### PR TITLE
Add playbook to migrate node imageConfig.format

### PR DESCRIPTION
**Pull request by michaelgugino**
There does not currently exist a way to migrate a node's
config imageConfig.format during transition between registries.

This commit adds a playbook that can be run on an as-needed
basis to update configmaps for each group in openshift_node_groups

### CODE REVIEW

**Comment by michaelgugino** [comment]
@sdodson @mtnbikenc @vrutkovs 

My current suggestion is to have users run this *after* upgrading all nodes to 3.11.  We could also add this in for upgrades to 3.12 to ensure they were migrated.

Essentially, we have a chicken and egg problem.  We can't do this before the upgrade because the new registry might not have the images, additionally, the hosts won't have the credentials yet.

We can't do it during the upgrade because it would cut all nodes over due to the sync pod.  This might cause failures due to the aforementioned reason.

We can't do this at the end of the upgrade_node playbook because users might have modified their inventories to only work on a subset of hosts, and this would update all configmaps in openshift_node_groupnames list.  We could probably work around this with 'yet another variable'.

**Comment by michaelgugino** [comment]
Hit failure on etcd health check: 'Verify cluster is healthy pre-upgrade' with the current patch set.

Seems restarting the node service bounces static pods :(

**Comment by michaelgugino** [comment]
Another failed task for web console: 'Ensure openshift-web-console project exists'

**Comment by sdodson** [comment]
If you run  playbooks/byo/openshift-cluster/upgrades/v3_11/upgrade_control_plane.yml multiple times we shouldn't observe any problems with part1 re-running should we? I can't think of any but wanted to make sure that playbook weren't forced in to running just once.

File: roles/openshift_node/tasks/upgrade/stop_services.yml [hunk 1]
```diff
@@ -7,16 +7,17 @@
...
   register: l_openshift_node_upgrade_docker_stop_result
   until: not (l_openshift_node_upgrade_docker_stop_result is failed)
   retries: 3
   delay: 30
-  when:
-  - l_docker_upgrade is defined
-  - l_docker_upgrade | bool
+  when: >
+        inventory_hostname in groups['oo_masters_to_config']
+        or (l_docker_upgrade is defined and l_docker_upgrade | bool)
```
Comment (sdodson) [inline roles/openshift_node/tasks/upgrade/stop_services.yml]: Honestly, I'd prefer we just stop and update docker everytime.

**Comment by michaelgugino** [comment]
> if you run playbooks/byo/openshift-cluster/upgrades/v3_11/upgrade_control_plane.yml multiple times we shouldn't observe any problems with part1 re-running should we? I can't think of any but wanted to make sure that playbook weren't forced in to running just once.

@sdodson I don't think we'd have a problem, just the node service will be bounced but that shouldn't kill any pods.

**Pull request merged**